frontend/taipy-gui/src/context/taipyReducers.ts

```diff
@@ -529,6 +552,37 @@ export const taipyReducer = (state: TaipyState, baseAction: TaipyBaseAction): Ta
             }
             return { ...state, data: data };
         }
+        case Types.Patch:
+            console.log("Patch", action);
+            const patchDesc = action as unknown as TaipyPatchAction;
+            if (patchDesc.names && patchDesc.names.length > 0 && (patchDesc.change || patchDesc.remove)) {
```

Comment:
I'm not sure I understand what `patchDesc.names.length > 1` would represent...
Applying the change/remove to different vars?

Reply:
applying the changes to different representation of the variable
for example a dict in a table and in graph ...